### PR TITLE
Bugfix/audit config defaults rc 1.13.0

### PR DESCRIPTION
**Pull request by bffrost**

### CODE REVIEW

File: modules/CmpdReg/src/client/custom/configuration.json [hunk 1]
```diff
@@ -1,112 +1,131 @@
...
+		"connectToServer": true,
+		"baseServerURL": "/cmpdreg/",
+		"logoutURL": "/logout",
+		"acasURL": "http://localhost:8080/acas/api/v1/",
+		"acasAppURL": "http://localhost:3001/api/"		 
+	},
+	"clientUILabels": {
+		"corpNameLabel": "CMPD Number",
+		"applicationNameForTitleBar": "ACAS Compound Registration",
+		"commonNameLabel": "Original ID:"
```
Comment (jmcneil86043) [inline modules/CmpdReg/src/client/custom/configuration.json]: I think "Common Name" would be better for default

File: modules/CmpdReg/src/client/custom/configuration.json [hunk 2]
```diff
@@ -1,112 +1,131 @@
...
+		"corpSeparator": "-",
+		"saltSeparator": "",	
+		"batchSeparator": "-",
+		"numberCorpDigits": 7,
+		"formatBatchDigits": 3,
+		"startingCorpNumber": 0,
+		"fancyCorpNumberFormat": false,
+		"corpParentFormat": "standard",
+		"corpBatchFormat": "corp_batch_saltcode",
+		"appendSaltCodeToLotName": true,
```
Comment (jmcneil86043) [inline modules/CmpdReg/src/client/custom/configuration.json]: let's not show the salt code in the lot name -> false

File: modules/CmpdReg/src/client/custom/configuration.json (hunk 1 above)
Comment (bffrost) [reply modules/CmpdReg/src/client/custom/configuration.json]: The actual UI field that uses this config was taken out when we added Aliases, so it is a dead config

Comment (jmcneil86043) [reply modules/CmpdReg/src/client/custom/configuration.json]: ok then

File: modules/CmpdReg/src/client/custom/configuration.json [hunk 3]
```diff
@@ -1,112 +1,135 @@
...
+				{"name": "Lot Salt Equivalents", "dataType": "string", "required": false},
+				{"name": "Lot Solution Amount Units", "dataType": "string", "required": false},
+				{"name": "Lot Solution Amount", "dataType": "numeric", "required": false},
+				{"name": "Lot Stock Location", "dataType": "string", "required": false},
+				{"name": "Lot Stock Solvent", "dataType": "string", "required": false},
+				{"name": "Lot Storage Location", "dataType": "string", "required": false},				
+				{"name": "Lot Supplier ID", "dataType": "string", "required": false},
+				{"name": "Lot Supplier Lot", "dataType": "string", "required": false},
+				{"name": "Lot Supplier", "dataType": "string", "required": false},
+				{"name": "Lot Synthesis Date", "dataType": "date", "required": false},
```
Comment (fkmeyer) [inline modules/CmpdReg/src/client/custom/configuration.json]: @bffrost don't we almost always have the date marked as required? Seems like it wouldn't be a bad default setting.

File: modules/CmpdReg/src/client/templates/AppControllerView.inc [hunk 4]
```diff
@@ -31,7 +31,7 @@
 
 	</div>
 
-	<div id="footer"><p>&#169; 2011-2015 John McNeil & Co., Inc.</p></div>
+	<div id="footer"><p>&#169; 2011-2017 John McNeil & Co., Inc.</p></div>
```
Comment (fkmeyer) [inline modules/CmpdReg/src/client/templates/AppControllerView.inc]: Should we go ahead and bump to 2018? @jmcneil86043 @bffrost

File: modules/ModuleMenus/src/client/ModuleMenusConfiguration.coffee [hunk 5]
```diff
@@ -31,12 +27,6 @@ window.ModuleMenusConfiguration =
...
 		requireUserRoles: [window.conf.roles.acas.userRole]
-	,
-		isHeader: false
-		menuName: "Project Editor"
-		mainControllerClassName: "ProjectController"
-		autoLaunchName:"project"
-		requireUserRoles: [window.conf.roles.acas.adminRole]
 	,
 		isHeader: false, menuName: "Example Thing"
 		mainControllerClassName: "ExampleThingController"
```
Comment (fkmeyer) [inline modules/ModuleMenus/src/client/ModuleMenusConfiguration.coffee]: Do we need to hide Example Thing?

File: modules/CmpdReg/src/client/custom/configuration.json (hunk 3 above)
Comment (bffrost) [reply modules/CmpdReg/src/client/custom/configuration.json]: We have had requests in the past to have the required set be only the set actually required by the database, so the most minimal set. This is why Lot Synthesis Date got changed (a couple versions ago) from required to not in the default bulk loader configs.

File: modules/ModuleMenus/src/client/ModuleMenusConfiguration.coffee (hunk 5 above)
Comment (bffrost) [reply modules/ModuleMenus/src/client/ModuleMenusConfiguration.coffee]: @jmcneil86043 ?

File: modules/CmpdReg/src/client/templates/AppControllerView.inc (hunk 4 above)
Comment (bffrost) [reply modules/CmpdReg/src/client/templates/AppControllerView.inc]: I defer to John on this one

File: modules/CmpdReg/src/client/custom/configuration.json (hunk 3 above)
Comment (fkmeyer) [reply modules/CmpdReg/src/client/custom/configuration.json]: Ah okay cool. I was thinking about it in the context of the single reg gui anyway. Makes sense!

**Pull request merged**